### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by AdityaPrakash-03** [Ijaiswalshivam/Talk-In#14] [Q&A]
![Screenshot 2024-05-10 234446](https://github.com/Ijaiswalshivam/Talk-In/assets/143942937/cd53c896-4c11-49ba-8644-3838141f0ea1)

can anyone help me with this ... how to run this ...always showing errors , what to do?

**Comment by Vishwajith-Shettigar** [comment]
You opened wrong directory in your android studio.

Open  **TalkIn** as root directory of this project.

![image](https://github.com/Ijaiswalshivam/Talk-In/assets/76042077/a2a5ffd8-597a-45d5-bf9a-54eba9dfb677)


step1: Go to file menu  in your android studio
step 2 :Click open
step 3: Locate this cloned project
step 4: Expand Talk-in
step 5: select TalkIn
step 6: press ok.

**Comment by Vishwajith-Shettigar** [comment]
Hope this helps.

**Comment by Ijaiswalshivam** [comment]
> 
> ![Screenshot 2024-05-10 234446](https://github.com/Ijaiswalshivam/Talk-In/assets/143942937/cd53c896-4c11-49ba-8644-3838141f0ea1)
> 
> can anyone help me with this ... how to run this ...always showing errors , what to do?

Fork again, because today some missed files are added
